src/Message/HostedCheckoutDecryptReturnUrlRequest.php

```diff
@@ -55,6 +55,7 @@ class HostedCheckoutDecryptReturnUrlRequest extends ExtendedAbstractRequest
 {
     /**
      * @return SimpleXMLElement
+     * @psalm-suppress PossiblyInvalidArrayAccess because the existence of the key is checked first before using it.
```

Comment:
wow it doesn't figure out the isset? i guess this is using an old psalm...

Reply:
Yeah this is happening on master so it's probably due to a new recent Pslam feature 😕